ui_framework/src/components/form/field_search/field_search.js

```diff
@@ -3,28 +3,50 @@ import React, {
 } from 'react';
 import classNames from 'classnames';
 
+import {
+  KuiFormControlLayout,
+} from '../form_control_layout';
 
-export const KuiFieldSearch = ({ className, id, name, placeholder, value, ...rest }) => {
+import {
+  KuiValidatableControl,
+} from '../validatable_control';
+
+export const KuiFieldSearch = ({
+  className,
+  id,
+  name,
+  placeholder,
+  value,
+  isInvalid,
+  ...rest,
+}) => {
   const classes = classNames('kuiFieldSearch', className);
 
   return (
-    <input
-      type="search"
-      id={id}
-      name={name}
-      placeholder={placeholder}
-      className={classes}
-      value={value}
-      {...rest}
-    />
+    <KuiFormControlLayout
```

Comment:
Like, I understood it, if you were just making it a fully naked input tag, but now it's got a necessary wrapper involved. At that point, just use the full wrapper and let people pass options?